FirebaseMLModelDownloader/Sources/CustomModel.swift

```diff
@@ -14,36 +14,10 @@
 
 import Foundation
 
-/// An enum of supported model formats.
-public enum CustomModelFormat {
-  case unknown
-  case tfLite
-  case coreML
-}
-
 /// A custom model that is stored remotely on the server and downloaded to the device.
-public struct CustomModel {
-  public let modelName: String
-  public var modelSize: Int
-  var modelPath: String
-  public var modelHash: String
-  public var modelFormat: CustomModelFormat
-
-
-  init(name: String, size: Int, path: String, hash: String, format: CustomModelFormat) {
-    modelName = name
-    modelSize = size
-    modelPath = path
-    modelHash = hash
-    modelFormat = format
-  }
-
-  public func getLatestModel() -> FileHandle? {
```

Comment:
We only need modelPath not modelFile according to the TFLite docs, so taking this out.